.github/workflows/pypipublish.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Python
+
+on:
+  release:
```

Comment:
```
push:
    branches:    
      - main
```
Would this be nice so it automatically runs on push to main? The trigger you have here runs on publishing the release notes if I understand correctly?

Reply:
I think it's better to do it on release, once we know for sure everything is correct. We obviously release on main, so I don't think it would make that big of a difference.